app/src/androidTest/java/com/sdp/movemeet/HomeScreen/HomeScreenActivityTest.java

```diff
@@ -1,44 +1,34 @@
-package com.sdp.movemeet;
+package com.sdp.movemeet.HomeScreen;
 
 import androidx.test.espresso.intent.Intents;
-import androidx.test.espresso.matcher.ViewMatchers;
 import androidx.test.ext.junit.rules.ActivityScenarioRule;
 import androidx.test.ext.junit.runners.AndroidJUnit4;
 
+import com.sdp.movemeet.HomeScreenActivity;
+import com.sdp.movemeet.R;
+
 import org.junit.Rule;
 import org.junit.Test;
 import org.junit.runner.RunWith;
 
 import static androidx.test.espresso.Espresso.onView;
 import static androidx.test.espresso.action.ViewActions.click;
-import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
-import static androidx.test.espresso.action.ViewActions.typeText;
-import static androidx.test.espresso.intent.Intents.intended;
-import static androidx.test.espresso.intent.matcher.IntentMatchers.hasExtra;
 import static androidx.test.espresso.matcher.ViewMatchers.withId;
 
-
 @RunWith(AndroidJUnit4.class)
 public class HomeScreenActivityTest {
 
     @Rule
-    public ActivityScenarioRule<HomeScreenActivity> testRule = new ActivityScenarioRule<>(HomeScreenActivity.class);
+    public ActivityScenarioRule<HomeScreenActivity> HomeScreenTestRule = new ActivityScenarioRule<>(HomeScreenActivity.class);
 
     @Test
     public void mainActivity_signIn() {
-        Intents.init();
-
         onView(withId(R.id.signInButton)).perform(click());
```

Comment:
We should probably check that the correct Intents are fired on click with an assertion

Reply:
Yes, in this case, the intent is created directly when the function is called via the click. We should also check that the click returns the right view.